elements/drag-drop-elements/pl-drag-drop-element.py

```diff
@@ -0,0 +1,163 @@
+import prairielearn as pl
+import lxml.html
```

Comment:
We will want to implement a new method here for the UI tests. 

`def test(element_html, data):`

You will want the test method to test for three scenarios:

- data['test_type'] == 'correct'
- data['test_type'] == 'incorrect'
- data['test_type'] == 'invalid'

You will want to take a look at some other elements to grab context, but you basically have to make sure that the logic of your element works consistently with PL's expectations under each of the three scenarios.

Reply:
Gotcha. Theres a few bugs I need to look at because of different jquery versions, but I should be able to get to adding tests soon now that the build tests are taken care of.

Reply:
Took a while but all the tests pass now :relieved: